### PR TITLE
[Term Entry] Adding Pytorch Tensor .tensor()

### LINKED ISSUE #4935: [Term Entry] PyTorch Tensors .tensor()
### Reason/inspiration (optional)

We would like a new entry on the `.tensor()` term under PyTorch `Tensors` concept. The entry should go in a new file under `docs/content/pytorch/concepts/tensors/terms/tensor/tensor.md`.

The entry should include:

- An introduction to the concept
- A ‘Syntax’ section that provides the syntax for the concept (if applicable)
- An ‘Example’ section that provides an example demonstrating the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by selenahuang017**
Description
Adding new entry on the .tensor() term under PyTorch Tensors concept. It is under docs/content/pytorch/concepts/tensors/terms/tensor/tensor.md.

Issue Solved
Closes #4935

Type of Change
Adding a new entry

Checklist
[x ] All writings are my own.
[x ] My entry follows the Codecademy Docs style guide.
[x ] My changes generate no new warnings.
[x ] I have performed a self-review of my own writing and code.
[x ] I have checked my entry and corrected any misspellings.
[x ] I have made corresponding changes to the documentation if needed.
[x ] I have confirmed my changes are not being pushed from my forked main branch.
[x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
[x ] I have linked any issues that are relevant to this PR in the Issues Solved section.

### CODE REVIEW

File: content/pytorch/concepts/tensors/terms/tensor/tensor.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - 'AI'
+  - 'Data Types'
+  - 'Deep Learning'
+  - 'Functions'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/data-science'
+---
+
+The **`.tensor()`** function creates a tensor from given data input.
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/tensor/tensor.md:17]:
```suggestion
The **`.tensor()`** function creates a tensor from a given data input.
```

File: content/pytorch/concepts/tensors/terms/tensor/tensor.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+Title: '.tensor()'
+Description: 'Creates a tensor of a given shape, and given values.'
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/tensor/tensor.md:3]:
```suggestion
Description: 'Creates a tensor of a given shape and given values.'
```

File: content/pytorch/concepts/tensors/terms/tensor/tensor.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+The **`.tensor()`** function creates a tensor from given data input.
+
+## Syntax
+
+```pseudo
+torch.tensor(data, dtype=None, device=None, requires_grad=False, pin_memory=False)
+```
+
+- `data`: This parameter is the input data for the tensor. The data must be array-like or scalar.
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/tensor/tensor.md:25]:
```suggestion
- `data`: This required parameter is the input data for the tensor. The data must be array-like or scalar.
```

File: content/pytorch/concepts/tensors/terms/tensor/tensor.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+The following example shows how to use the `.tensor()` function:
+
+```py
+import torch
+
+t1 = torch.tensor(3)
+print(t1)
+
+t2 = torch.tensor([1.0, 3.0, 5.0], [2.0, 6.0, 0.0])
+print(t2)
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/tensor/tensor.md:42]: Let's change the code to get the desired output:

```suggestion
import torch

t1 = torch.tensor(3)
print(t1, '\n')

t2 = torch.tensor([[1.0, 3.0, 5.0], [2.0, 6.0, 0.0]])
print(t2)
```

**Review by Sriparno08 — CHANGES_REQUESTED** [review]
Hey @selenahuang017, thanks for contributing to Codecademy Docs!

I've listed all the modifications that you need to make in your entry below:

**Review by Sriparno08 — APPROVED** [review]
Good job, @selenahuang017!

This entry looks good for a second round of review. 👍

**Review by mamtawardhani — APPROVED** [review]
Thank you for your contribution @selenahuang017 😄 

The entry looks good to be merged! 🚀
